surveys/2024-annual-survey/questions.md

```diff
@@ -161,7 +161,8 @@ Type: select all that apply (optional)
 - Windows 10/11
 - Windows 8 or older
 - Windows Subsystem for Linux
-- macOS
+- macOS (x64)
```

Comment:
does apple call them like that too? I thought "x64" was mostly a Windows thing.
but either way I think this should elaborate and probably say something like "macOS (Intel)", "macOS (M-series Apple Silicon)" or something like that, to make it clear what this means. not everyone is familiar with the technical name of their CPU architecture

Reply:
Yeah, I was a bit lazy with the name :laughing: Changed it to `Intel`, that should be universally understandable, I hope.